crypto-serde/src/slice.rs

```diff
@@ -41,6 +40,68 @@ where
     }
 }
 
+/// Deserialize the given slice from hex when using human-readable formats or
+/// binary if the format is binary.
+pub fn deserialize_hex_or_bin_with_memory<'de, D>(
```

Comment:
In several of the other crates in this repo we use a `deserialize_*` vs `deserialize_*_vec` to distinguish the mutating vs allocating APIs. I'd recommend that here.

Reply:
Done.